packages/store/src/operators/of-action.ts

```diff
@@ -74,6 +74,40 @@ export function ofActionErrored(...allowedTypes: any[]) {
   return ofActionOperator(allowedTypes, [ActionStatus.Errored]);
 }
 
+function mapToActionExecuting(allowedType: any) {
```

Comment:
1) move to another file '../utils/utils'
2) function mapToActionExecuting(allowedType: ActionType) {